extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1986,6 +1986,41 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'custom': 'https://${apiEndpoint}/event/pixel${base}&collection=${collection}',
     },
   },
+  
+ 'catchpoint': {
+	
+    'requests': {	
+      'endpoint': 'https://r.3gl.net/hawklogserver/r.p', 
+      'pageview': 'https://r.3gl.net/hawklogserver/r.p' +
+		'?'+	'data={"v":2,"y":"load","ui":${UserID},"si":${SessionID},"pi":092934,"di":${AccountID},"pc":1,"dn":${dns},"fc":7,"wt":${wait},"Id":${load},"de":${domInteractive},"dl":${domLoaded},"dc":${docComplete},"rp":1338,"cl":${contentLoad},"dc":${docComplete},"rp":${response},"cl":${contentLoad},"rd":0,"jsc":0,"dh":1200,"dw":1920,"el":193,"rf":"${canonicalUrl}","cv":${isConversion},"rv":${revenue},"ri":${revenue_items}}'
```

Comment:
80 chars limit please

Reply:
vendor-requests.json & vendor.js files are updated

Reply:
I didn't see the change.  Please use `+` to separate the line to multiple strings. Thanks.

Also please add test to extensions/amp-analytics/0.1/test/vendor-requests.json  https://github.com/ampproject/amphtml/pull/16171#pullrequestreview-132190636